spec/02-integration/05-proxy/02-router_spec.lua

```diff
@@ -130,6 +131,7 @@ for _, strategy in helpers.each_strategy() do
     local proxy_ssl_client
     local bp
     local it_trad_only = (flavor == "traditional") and it or pending
+    local it_not_trad = (flavor ~= "traditional") and it or pending
```

Comment:
I think that we need not to define `it_not_trad`, 
just use `if flavor ~= "traditional" then end` around those special cases.